posthog/async_migrations/migrations/0010_minmax_indexes_for_materialized_columns.py

```diff
@@ -0,0 +1,72 @@
+from functools import cached_property, partial
```

Comment:
could we add a test checking that given some existing indexes, we actually pull the right columns to index, and issue the correct DDL?